client/src/app/ui/modules/global-headbar/modules/account/components/account-dialog/account-dialog.component.ts

```diff
@@ -144,7 +165,6 @@ export class AccountDialogComponent extends BaseUiComponent implements OnInit {
     }
 
     private async updateIsUserInScope(): Promise<void> {
-        this._isUserInScope = true;
-        // this._isUserInScope = await this.repo.isUserInSameScope(this.operator.operatorId);
+        this._isUserInScope = await this.userService.isUserInSameScope(this.operator.operatorId);
```

Comment:
This seemed like an accidental regression @GabrielInTheWorld - please correct me if I'm wrong